Module4/2_Scope/1.Function's_scope.py

```diff
@@ -0,0 +1,28 @@
+"""Function's Scope"""
+print("Welcome Function's Scope")
+
+'''1.You can't see a variable inside a function'''
+def scope1():
+    var1=1
+
+try:
```

Comment:
there are a couple of these code, don't worry if you don't understand them now, I'm going to explain me later 😺